src/content/en/updates/2019/06/layoutNG.md

```diff
@@ -0,0 +1,328 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: A new layout engine for Chromium
+
+{# wf_published_on: 2019-06-06 #}
+{# wf_updated_on: 2019-06-06 #}
+{# wf_featured_image: /web/updates/images/2019/06/layoutNG-header.jpg #}
+{# wf_tags: new-in-chrome,layout,performance #}
+{# wf_featured_snippet: LayoutNG is a new layout engine for Chromium that has been designed for the needs of modern scalable web applications. It improves performance isolation, better supports scripts other than Latin, and fixes many float, margin, and web compatibility issues. #}
+{# wf_blink_components: N/A #}
+
+# LayoutNG {: .page-title }
+
+{% include "web/_shared/contributors/emilaeklund.html" %}
+
+<style>
+  .w-figures-2-up {
+    display: flex;
+    justify-content: space-around;
+    align-items: flex-start;
+    display: grid;
+    gap: 1rem;
+    grid-template-columns: repeat(auto-fit, minmax(20ch, 1fr));
+    margin: 3rem 0;
+  }
+
+  .w-figures-2-up figure {
+    text-align: center;
+  }
+</style>
+
+Scheduled to be released in Chrome 76, LayoutNG is a new layout engine exiting a 
+multi-year effort. There are several exciting immediate improvements, 
+and additional performance gains and advanced layout features will be coming.
+
+## What's New?
+
+1. Improves performance isolation.
+1. **Better support** for scripts other than Latin
+1. **Fixes** many issues around floats and margins
+1. **Fixes** a large number of web compatibility issues
+
+Please note that LayoutNG will be **launched in stages**. In Chrome 76, LayoutNG 
+is used for inline and block layout. Other layout primitives 
+(such as table, flexbox, grid, and block fragmentation) will be replaced in subsequent releases.
+
+### Developer visible changes
```

Comment:
Total nit - leave a blank line between headers and the content.

Reply:
👍